cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/events/LanguageEventData.java

```diff
@@ -48,6 +48,10 @@
     public static final String STEP_TYPE = "STEP_TYPE";
     public static final String STEP_NAME = "STEP_NAME";
     public static final String CONTEXT = "CONTEXT";
+    public static final String CONSUMER_WORKER_UUID = "CONSUMER_WORKER_UUID";
+    public static final String PRODUCER_WORKER_UUID = "PRODUCER_WORKER_UUID";
+    public static final String ROBOT_UUID = "ROBOT_UUID";
+    public static final String ACTIVITY_GROUP = "ACTIVITY_GROUP";
```

Comment:
Think about renaming to ROBOT_GROUP_NAME to keep consistency with WORKER_GROUP_NAME